music21/serial.py

```diff
@@ -1059,27 +1059,12 @@ def areCombinatorial(self, transType1, index1, transType2, index2, convention):
         if self.isTwelveToneRow() is False:
             raise SerialException('Combinatoriality applies only to twelve-tone rows.')
 
-        if convention == 'zero':
-            testRow = []
-            trans1 = self.zeroCenteredTransformation(transType1, index1)
-            pitches1 = trans1.pitchClasses()
-            trans2 = self.zeroCenteredTransformation(transType2, index2)
-            pitches2 = trans2.pitchClasses()
-            for i in range(6):
-                testRow.append(pitches1[i])
-            for i in range(6):
-                testRow.append(pitches2[i])
-            return pcToToneRow(testRow).isTwelveToneRow()
-        elif convention == 'original':
-            testRow = []
-            trans1 = self.originalCenteredTransformation(transType1, index1)
+        if convention in ['zero', 'original']:
+            trans1 = getattr(self, convention + 'CenteredTransformation')(transType1, index1)
             pitches1 = trans1.pitchClasses()
-            trans2 = self.originalCenteredTransformation(transType2, index2)
+            trans2 = getattr(self, convention + 'CenteredTransformation')(transType2, index2)
             pitches2 = trans2.pitchClasses()
-            for i in range(6):
-                testRow.append(pitches1[i])
-            for i in range(6):
-                testRow.append(pitches2[6 + i])
+            testRow = pitches1[:6] + pitches2[:6]
```

Comment:
Hi Mark -- is there a music-theoretical reason this is equivalent to `pitches2[6:]` above for `original`? It's been a while ...

Reply:
Indeed, the computation is different now. Print the `testRow` and run the second example in the doctest:

master:
[5, 4, 1, 11, 10, 0, 11, 9, 10, 8, 2, 3]
PR:
[5, 4, 1, 11, 10, 0, 0, 1, 4, 6, 7, 5]

Reply:
Thanks for flagging this up. I've responded in the summary below.